File: Iniciante/URI 1180.py
# https://www.beecrowd.com.br/judge/pt/problems/view/1180

n = int(input())
vetor = list(map(int, input().split()))
menor = min(vetor)
index = vetor.index(menor)
print (f"Menor valor: {menor}")
print (f"Posicao: {index}")

File: Iniciante/URI 1180.cpp
// https://www.beecrowd.com.br/judge/pt/problems/view/1180

#include <iostream>
using namespace std;
int main()
{
    int n; cin >> n;
    int vetor[n];
    for(int i=0; i < n; i++)
    {
        cin >> vetor[i];
    }
    int min = vetor[0], index=0;
        for(int i=0; i < n; i++)
    {
        if (vetor[i] < min)
        {
            min = vetor[i];
            index = i;
        }
    }
    cout << "Menor valor: " << min << endl;
    cout << "Posicao: " << index << endl;
}